src/App.js

```diff
@@ -1,23 +1,90 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import contactsData from "../src/contacts.json";
+import "./App.css";
 
 function App() {
+  const [contacts, setContacts] = useState(contactsData.slice(0, 5));
+
+  const randomContacts = () => {
```

Comment:
A good idea here is to apply Math.random only on the remaining contacts on the list.
The way you did you might get repeated contacts.